src/test/java/com/kodilla/ecommercee/domain/GroupRepositoryTestSuite.java

```diff
@@ -0,0 +1,145 @@
+package com.kodilla.ecommercee.domain;
+
+import com.kodilla.ecommercee.repository.GroupRepository;
+import com.kodilla.ecommercee.repository.ProductRepository;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class GroupRepositoryTestSuite {
+    @Autowired
+    private ProductRepository productRepository;
+
+    @Autowired
+    private GroupRepository groupRepository;
+
+    private Group group1;
+    private Group group2;
+    private Product product1;
+    private Product product2;
+    private Product product3;
+    private Product product4;
+
+    @Test
+    public void testGroupRepositorySave() {
+        //Given
+        createData();
+
+        //When
+        groupRepository.save(group1);
```

Comment:
proponuje wynik tej metody przypisać do `Group result` a następnie to z niej wyciągnąć ID w asercji. Wtedy pokazujesz że rzeczywiście ta metoda działa

Reply:
Próbowałem tak zrobić, ale przy testach w asercji raz actual value jest równe 1, a raz 27, więc nie mogę ustalić  prawidłowego expected value, dlatego zostawię tak jak było.

Reply:
Ale asercję zostawiasz w takiej postaci jak jest, czyli sprawdzasz czy ID nie jest równe 0. W związku z tym że miałeś 1 lub 27 to asercja się spełni. Chodzi mi tylko do czego porównujesz 0.

Reply:
Okej